ui/src/app/views/workspace/component.ts

```diff
@@ -283,6 +292,38 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
     this.router.navigate(['share'], {relativeTo : this.route});
   }
 
+  sendANotebook(notebook): void {
```

Comment:
nit: I would probably name this `sendNotebook` instead. Or `selectNotebook` per below.